sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/implementation/AadRestOperationConfiguration.java

```diff
@@ -29,51 +33,77 @@
 public class AadRestOperationConfiguration {
 
     /**
-     * Declare {@link RestOperations} bean which will be used when send http request to Azure AD and Azure AD B2C.
-     * <br/>
-     * For example:
+     * Bean name of RestOperation bean. The bean is used to retrieve access token.
+     *
+     * @see AadRestOperationConfiguration#azureAdAccessTokenRetrieverRestOperations(RestTemplateBuilder)
+     */
+    public static final String AZURE_AD_ACCESS_TOKEN_RETRIEVER_REST_OPERATIONS_BEAN_NAME =
+            "azureAdAccessTokenRetrieverRestOperations";
+
+    /**
+     * Bean name of RestOperation bean. The bean is used to get key to validate access token.
+     *
+     * @see AadRestOperationConfiguration#azureAdAccessTokenValidatorRestOperations(RestTemplateBuilder)
+     */
+    public static final String AZURE_AD_ACCESS_TOKEN_VALIDATOR_REST_OPERATIONS_BEAN_NAME =
+            "azureAdAccessTokenValidatorRestOperations";
+
```

Comment:
Better put these bean names in `AadResourceServerConfiguration`, and we better call them by wider scenario, like `oauth2 client` and `resource server`